openslides_backend/shared/mixins/user_create_update_permissions_mixin.py

```diff
@@ -203,20 +206,24 @@ class CreateUpdatePermissionsMixin(UserMixin, UserScopeMixin, Action):
             "is_present",  # participant import
         ],
         "C": ["meeting_id", "group_ids"],
-        "D": ["committee_ids", "committee_management_ids"],
+        "D": ["committee_management_ids"],
         "E": ["organization_management_level"],
         "F": ["default_password"],
         "G": ["is_demo_user"],
         "H": ["saml_id"],
     }
 
-    def check_permissions(self, instance: dict[str, Any]) -> None:
+    def check_permissions(self, instance: dict[str, Any]) -> bool | None:
```

Comment:
This function always returns `True` or Error, never `False`.
Having it be a boolean return type is therefore unnecessary and only removes it from the action check_permissions functions which it is supposed to implement.
Remove the return type `bool`.